SpatialGDK/Documentation/content/get-started/gdk-and-starter-project.md

```diff
@@ -0,0 +1,63 @@
+
+**Note** You need to ensure that the root folder of the Unreal GDK repository is called `UnrealGDK` so its path is something like: `<StarterProjectRepoRoot>\Game\Plugins\UnrealGDK\...`
+
+[//]: # (TODO: This whole section below deserves some screenshots to show users what to expect. This was done well on the Unity onobarding docs. Example: https://docs.improbable.io/unity/alpha/content/get-started/get-playing)	
+
+### Build Dependencies 
+
+In this step, you're going to build the Unreal GDK module dependencies, which the Starter Project needs to work with the GDK. You are then going to add the Unreal GDK to the Starter Project.
+
+1. Open **File Explorer**, navigate to the root directory of the Unreal GDK repository, and double-click **`Setup.bat`**. You may be prompted to sign into your SpatialOS account if you have not already.
+2. Set the Starter Project to work with the Unreal Engine fork you cloned and installed earlier. To do this:
+    1. In File Explorer, navigate to the root directory of the Unreal GDK Starter Project repository, and then to the **Game** directory within it.
+    2. Right-click **StarterProject.uproject** and select **Switch Unreal Engine version**.
+    3. Select the path to the Unreal Engine fork you cloned earlier.
+3. In the same directory, open **StarterProject.sln** with Visual Studio.
+4. In the Solution Explorer window, right-click on **StarterProject** and select **Build**.
+5. Open **StarterProject.uproject** in the Unreal Editor and click [`Schema` (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary) to generate schema and then [`Snapshot` (SpatialOS documentation)]((https://docs.improbable.io/reference/latest/shared/glossary)) to generate a snapshot.
+
+### Deploy the Starter Project locally
+
+1. In the Unreal Editor, on the SpatialOS GDK toolbar, click **Launch**. Wait until you see the output `SpatialOS ready. Access the inspector at http://localhost:21000/inspector`.
+2. On the Unreal Editor toolbar, open the **Play** drop-down menu.
+3. Under **Multiplayer Options**, enter the number of players as **2** and check the box next to **Run Dedicated Server**. Then, under Modes, select **New Editor Window (PIE)**.
+4. On the toolbar, click **Play** to run the game.
+5. When you're done, click **Stop** to stop the client [worker](https://docs.improbable.io/reference/latest/shared/glossary), and click the **Stop** button in the SpatialOS section of the toolbar to stop your local SpatialOS deployment.
+
+### Deploy the Starter Project in the cloud
+
+To run a cloud deployment, you need to prepare your server-worker and client-worker [assemblies](https://docs.improbable.io/reference/latest/shared/glossary), and upload them to the cloud.
+
+> Building the assemblies can take a while - we recommend installing [IncrediBuild](https://www.incredibuild.com/), [FastBuild](http://www.fastbuild.org/), or another build distributor.
```

Comment:
Are we still advising Inredibuild, etc?
Also, you can't really use FastBuild with Unreal 4.20 right now.

Reply:
I removed fastbuild suggestions
Incredibuild I don't know - @girayimprobable ?